code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1800,4 +1817,22 @@
 		var/datum/antagonist/changeling/changeling = L.mind.has_antag_datum(/datum/antagonist/changeling)
 		if(changeling)
 			changeling.chem_charges = max(changeling.chem_charges-2, 0)
-	return ..()
\ No newline at end of file
+	return ..()
+
+/datum/reagent/kindleium //works like if you get hit by kindle
+	name = "Kindleium"
+	id = "kindleium"
+	description = "Supposedly refined from the remains of a fallen god."
+	color = "#ffe74c" // rgb: 255, 231, 76
+	taste_description = "spinning gears"
+
+/datum/reagent/kindleium/on_mob_add(mob/living/M)
+	kindle(M)
+
+/datum/reagent/kindleium/on_mob_life(mob/living/M)
+	if(prob(5) && !(M.has_status_effect(STATUS_EFFECT_KINDLE)))
```

Comment:
If you don't want the check just make kindleium without the check then a subtype with the check for the plants.

Reply:
using the reagent for the normal kindle projectile is probably bad, see the other review

Reply:
wait so what are you trying to tell the game to do here? Right now it's a prob(5) to kindle if the person doesn't have the status effect but otherwise do nothing.

Reply:
Exactly that. The intent was that it kindles you when you first eat it, but it'd be a bit strange for dosage to be completely irrelevant after that wears off so it will also occasionally kindle you while its in your system. It checks to make sure you aren't _already_ kindled so that you can't be chainstunned unless you're immensely unlucky.

Reply:
ok i've wrapped my moisturized brain around it thank you